src/OpenTelemetry.Exporter.OpenTelemetryProtocol.Logs/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+## 1.5.1
```

Comment:
We don't have to release this package.

Reply:
Same for both the Prometheus exporter packages.

Reply:
Updated